libs/libxml2/Makefile

```diff
@@ -100,6 +100,7 @@ CONFIGURE_ARGS += \
 	--with-xpath \
 	--with-xptr \
 	--with-zlib=$(STAGING_DIR)/usr \
+	--with-iconv="$(ICONV_PREFIX)" \
```

Comment:
In case `CONFIG_BUILD_NLS` is n (unset, or in other words "Compile with full language support" is not selected), then `$ICONV_PREFIX` is empty string and thus this line results in wrong syntax.
I think we must use:
`--with-iconv$(if $(ICONV_PREFIX),="$(ICONV_PREFIX)")`

Reply:
From what I see in current include/nls.mk, $(ICONV_PREFIX) is never empty:

ifeq ($(CONFIG_BUILD_NLS),y)
	ICONV_PREFIX:=$(STAGING_DIR)/usr/lib/libiconv-full
	ICONV_FULL:=1

	INTL_PREFIX:=$(STAGING_DIR)/usr/lib/libintl-full
	INTL_FULL:=1

# iconv stub
else
	ICONV_PREFIX:=$(STAGING_DIR)/usr/lib/libiconv-stub
	ICONV_FULL:=

	INTL_PREFIX:=$(STAGING_DIR)/usr/lib/libintl-stub
	INTL_FULL:=
endif

Reply:
will not be true soon.

Reply:
Would $(STAGING_DIR)/usr/lib/libiconv.so be available when $(ICONV_PREFIX) is empty?

Reply:
No.